contribs/gmf/src/services/treemanager.js

```diff
@@ -13,8 +13,8 @@ goog.require('ngeo.StateManager');
  * 'add', children will be just pushed in this array. The default state can be
  * changed by setting the value `gmfTreeManagerModeFlush`, e.g.:
  *
- *    var module = angular.module('app');
- *    module.value('gmfTreeManagerModeFlush', false);
+ *     var module = angular.module('app');
+ *     module.value('gmfTreeManagerModeFlush', false);
```

Comment:
Between the two adds (desktop / desktop_alt) we also have some other changes:
```
            <a href class="btn btn-default btn-block btn-primary"
              data-toggle="dropdown">
              <span class="fa fa-grid"></span>
              <span translate>Theme:</span>
              <b ng-if="!mainCtrl.gmfThemeManager.themeName" translate>Loading...</b>
              <b ng-if="mainCtrl.gmfThemeManager.themeName">{{mainCtrl.gmfThemeManager.themeName|translate}}</b>
              <span class="caret"></span>
            </a>
```
=>
```
            <a href class="btn btn-default btn-block btn-primary"
              data-toggle="dropdown">
              <span class="fa fa-grid"></span>
              <span translate>Themes</span>
              <span class="caret"></span>
            </a>
```